civiwiki/api/models/article.py

```diff
@@ -2,6 +2,15 @@
 from django.db import models
 
 # Create your models here.
+class ArticleManager(models.Manager):
+   def serialize(self, account, filter=None):
```

Comment:
If you look at my code in **account.py** carefully, you will notice that the data is being retrieved from the variable being passed into the function. Not the model objects. 
![image](https://cloud.githubusercontent.com/assets/6856645/14410922/50d42664-ff00-11e5-828a-5fac83867a2b.png)

if i were to do:
`data = { "email": Account.email }`
the code would break because the **Account** object has no attribute called email, only the  **account** variable that we pass into the function which is an instance of an Account object and not the class itself.